src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -368,7 +368,7 @@
          Also, we are using the "Exec" task to invoke MSBuild.exe rather the "MSBuild" task when building
          DownloadSonarPlugin.csproj to avoid getting file-locking errors when rebuilding.
     -->
-    <Exec Command="&quot;$(MSBuildBinPath)\msbuild.exe&quot; ..\..\build\DownloadSonarPlugin\DownloadSonarPlugin.csproj /p:PluginUrl=$(PluginUrl) /p:TargetDirectory=&quot;$(CFamilyTargetDirectory)&quot; /p:UseSharedCompilation=false /t:Restore;Build;EmbedCFamilyFiles" LogStandardErrorAsError="true" IgnoreExitCode="false" />
+    <Exec Command="&quot;$(MSBuildBinPath)\msbuild.exe&quot; ..\..\build\DownloadSonarPlugin\DownloadSonarPlugin.csproj /p:PluginUrl=$(PluginUrl) /p:TargetDirectory=&quot;$(CFamilyTargetDirectory)&quot; /p:UseSharedCompilation=false /t:Restore;Build;EmbedCFamilyFiles /nr:false" LogStandardErrorAsError="true" IgnoreExitCode="false" />
```

Comment:
I'm not sure if this would actually help, a few [threads](https://stackoverflow.com/questions/3919892/msbuild-exe-staying-open-locking-files) on stackoverflow recommended adding this flag. But I'm not sure _why_ it happens to begin with: according to [this](https://stackoverflow.com/a/3405932) post it shouldn't have happened.

Reply:
It does seem odd, but there's no harm in trying it. If it helps, great, and if we had time we could then try to work out why.